atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/RangeRequests.java

```diff
@@ -20,7 +20,10 @@
 import com.google.common.base.Preconditions;
 import com.palantir.atlasdb.encoding.PtBytes;
 
-public class RangeRequests {
+public final class RangeRequests {
```

Comment:
It makes no sense to instantiate classes that only have static methods, therefore it needs the private constructor, therefore it needs to be final.